README.md

```diff
@@ -96,6 +96,36 @@ UI.
 Conduit can be configured through command line parameters. To view the full list
 of available options, run `./conduit --help`.
 
+### Homebrew
+
+Make sure you have [homebrew](https://brew.sh/) installed on your machine, then run:
+
+```sh
+brew install conduit
+brew upgrade conduit
+```
+
+### Debian
+
+Download the right `.deb` file for your machine architecture from the
+[latest release](https://github.com/conduitio/conduit/releases/latest), then run:
+
+```sh
+apt-get install dpkg
+dpkg -i package_name.deb # replace with the downloaded package name
```

Comment:
What if we use the filename of the latest release, i.e. `conduit_0.6.0_Linux_x86_64.deb`? Then I think it's clear enough that we don't need the comment. Same for the RPM example.